tests/Application/config/packages/security_checker.yaml

```diff
@@ -1,9 +1,9 @@
-services:
-    SensioLabs\Security\SecurityChecker:
-        public: false
-
-    SensioLabs\Security\Command\SecurityCheckerCommand:
-        arguments: ['@SensioLabs\Security\SecurityChecker']
-        public: false
-        tags:
-            - { name: console.command, command: 'security:check' }
+#services:
```

Comment:
Why did you comment this out instead of removing this file?

Reply:
I forgot to remove it nice catch.